src/com/ableton/VirtualEnv.groovy

```diff
@@ -25,6 +27,11 @@ class VirtualEnv implements Serializable {
   }
 
 
+  def create() {
+    script.sh("virtualenv --python=${python} ${destDir}")
+  }
+
+
```

Comment:
Unclear about the spacing here, it's 2-per-def here and 1-per-@test below. Assuming this is standard groovy, just wanted to check.

Reply:
Standard Groovy doesn't make any recommendations on vertical whitespacing. [Google's Java style guide](https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace) says:

> Multiple consecutive blank lines are permitted, but never required (or encouraged).

@ala-ableton encouraged the adoption of 2 newlines between methods, which I guess matches our C++ style guidelines. I would personally rather just go with 1 newline everywhere, but actually, this discussion should move to https://github.com/AbletonAppDev/guides/pull/55 instead. Let's sort out the style issues there, and then go through our Groovy code afterwards to make it compliant (which we'll pretty much have to do anyways).

Reply:
The rule I wanted to adopt is the Python rule. 2 newlines between top-level things (classes, functions, ...) and 1 newline between nested things (methods, ...). So there should be only 1 empty line in this case.

Reply:
Ok, I'll fix this (and the other parts of the code) accordingly, and also update the style guide with that guideline.